Part3

Valaubr opened the pull request:
Задачи перого и второго урока.

https://www.e-olymp.com/ru/submissions/5739145 
 https://www.e-olymp.com/ru/submissions/5852076 
https://www.e-olymp.com/ru/submissions/5838840 
https://www.e-olymp.com/ru/submissions/5758661 
 https://www.e-olymp.com/ru/submissions/5855012 
 https://www.e-olymp.com/ru/submissions/5852022 
https://www.e-olymp.com/ru/submissions/5865002 
https://www.e-olymp.com/ru/submissions/5840331

Valaubr committed:
https://www.e-olymp.com/ru/submissions/5739145 && https://www.e-olymp.com/ru/submissions/5852076 && https://www.e-olymp.com/ru/submissions/5838840 && https://www.e-olymp.com/ru/submissions/5758661 && https://www.e-olymp.com/ru/submissions/5855012 && https://www.e-olymp.com/ru/submissions/5852022 && https://www.e-olymp.com/ru/submissions/5840203 && https://www.e-olymp.com/ru/submissions/5840331

Valaubr committed:
https://www.e-olymp.com/ru/submissions/5739145 && https://www.e-olymp.com/ru/submissions/5852076 && https://www.e-olymp.com/ru/submissions/5838840 && https://www.e-olymp.com/ru/submissions/5758661 && https://www.e-olymp.com/ru/submissions/5855012 && https://www.e-olymp.com/ru/submissions/5852022 && https://www.e-olymp.com/ru/submissions/5840203 && https://www.e-olymp.com/ru/submissions/5840331

dormidon commented:
Тут https://www.e-olymp.com/ru/submissions/5855012 и тут https://www.e-olymp.com/ru/submissions/5852022 не все тесты проходят.
По-хорошему надо бы добить :)

dormidon commented:
Принято частично - по 8 баллов.

Valaubr committed:
https://www.e-olymp.com/ru/problems/3738
https://www.e-olymp.com/ru/problems/1462
https://www.e-olymp.com/ru/problems/4741
https://www.e-olymp.com/ru/problems/4827
https://www.e-olymp.com/ru/problems/4037 -> (Пометочка:
Я решил немного поэксперементировать и найти другой путь решения задачи "Сортировка слиянием", но видимо с успехом провалился :D)

dormidon commented on src/main/java/ru/mail/polis/ads/dz3/DZ3_First.java:
```diff
@@ -0,0 +1,87 @@
+
+        int i = left, j = right;
+        while (i <= j) {
+            while (mass[i] < opora) {
```
O gospodi. Zachem zhe tak delat? :)
